### DISCUSSION TITLE
Repository not updated despite my push in the terminal

### DISCUSSION THREAD

**Discussion by LizzyKoon** [community/community#69593] [New to GitHub] [answered]
Hello! 
I duplicated my workspace and deleted the original. I created branches and I push them on Github but it doesn't appear even if the terminal tells me "everything is up-to-date". I see the branches in the terminal and VS Code but not in github (even if the name of the repo, the url path, etc are the same). What can I do? 
Thanks!

**Comment by moayad-star** [comment, marked as answer]
Open terminal :
--> git push remote_name barnch_name
_Like : git push origin div_

**replace : div ---> your branch name
replace : origin --> your remote name [Often: origin]**

**Please, if the problem is solved, mark my answer as “answered” to help the other people  solve a similar problem.**

**Reply by LizzyKoon** [reply]
Thanks a lot! 

Actually, I **created a new repository on Github**, and I followed your recommendation. 

- I replaced origin by the name of my new repository
- and I replaced div by the name of the branch of the old repository I wanted to move to the new one

**Reply by moayad-star** [reply]
I'm glad it's working well, 
if you have any other questions about git or GitHub, create a discussion and feel free to tag me.❤️👨‍💻
